code/game/machinery/flasher.dm

```diff
@@ -37,14 +37,13 @@
 //		src.sd_SetLuminosity(0)
 
 //Don't want to render prison breaks impossible
-/obj/machinery/flasher/attackby(obj/item/W as obj, mob/user as mob)
-	if (iswirecutter(W))
-		add_fingerprint(user)
-		src.disable = !src.disable
-		if (src.disable)
-			user.visible_message("<span class='warning'> [user] has disconnected the [src]'s flashbulb!</span>", "<span class='warning'> You disconnect the [src]'s flashbulb!</span>")
-		if (!src.disable)
-			user.visible_message("<span class='warning'> [user] has connected the [src]'s flashbulb!</span>", "<span class='warning'> You connect the [src]'s flashbulb!</span>")
+/obj/machinery/flasher/wirecutter_act(mob/living/user, obj/item/W)
```

Comment:
I assume this was supposed to be in your other PR

Reply:
This was a more generic misc related and was meant to be here.

Reply:
Not really proper atomization but meh, it's a small PR